src/Shared/Internal/RendererReEntrantManager.cs

```diff
@@ -68,19 +71,17 @@ private void Unlock()
         private bool _obtainedLock;
```

Comment:
Can this become `private readonly bool _obtainedLock` and assigned in the constructor ? (Return value from `TryGetLock()`)

Reply:
Complete.